src/test/kotlin/org/rust/ide/console/RsConsoleCompletionTestBase.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.console
+
+import org.rust.ProjectDescriptor
+import org.rust.WithStdlibRustProjectDescriptor
+import org.rust.lang.core.completion.RsCompletionTestBase
+import org.rust.lang.core.psi.RsFile
+import org.rust.lang.core.psi.RsReplCodeFragment
+
+@ProjectDescriptor(WithStdlibRustProjectDescriptor::class)
+abstract class RsConsoleCompletionTestBase : RsCompletionTestBase() {
+
+    protected fun checkReplCompletion(previousLines: String, lastLine: String, variant: String) {
+        checkReplCompletion(previousLines, lastLine, arrayOf(variant))
+    }
+
+    protected fun checkReplCompletion(previousLinesOriginal: String, lastLineOriginal: String, variants: Array<String>) {
+        val previousLines = previousLinesOriginal.trimIndent()
+        val lastLine = lastLineOriginal.trimIndent()
+
+        doCheckReplCompletion(previousLines, lastLine, variants)
+        if (previousLines.isBlank()) return
+
+        val lastLines = previousLines + lastLine
+        doCheckReplCompletion("", lastLines, variants)
+        doCheckReplCompletion("", "if true { $lastLines }", variants)
+        doCheckReplCompletion("", "fn main() { $lastLines }", variants)
+    }
+
+    private fun doCheckReplCompletion(previousLines: String, lastLine: String, variants: Array<String>) {
+        inlineReplCodeFragment(lastLine, previousLines)
+
+        val lookups = myFixture.completeBasic()
+        checkNotNull(lookups) {
```

Comment:
Note, `myFixture.completeBasic()` can return `null` when there is only one completion variant

Reply:
Thank you, I replaced all tests which expect one completion variant to use `RsCompletionTestFixture::executeSoloCompletion`